routing_model/index.html

```diff
@@ -342,6 +336,19 @@ <h2>Relocated node connection</h2>
     <p>Return true if the given message_src of the current RPC is a one of our nodes and is in WaitingConnectionInfo state.
     </p>
   </div>
+  <button class="collapsible">shorter_prefix_section</button>
+  <div class="content">
+    <p>If we know of a section that has a shorter prefix than ours, we prefer for them to receive this incoming node rather than ourselves as it will help keep the Network's sections tree balanced.<br/>
+    This shorter_prefix_section is a function that will return None if we are the shortest of any section we know, Some if there is a better candidate.<br/>
+    If it is Some, we will relay the RPC to them instead of accepting the rellocation to our own section.
+    </p>
+  </div>
+  <button class="collapsible">RelocatingState(Hop)</button>
```

Comment:
I don't think this name is consistent with other ones. Maybe the state could be: `RelocatingToShorterSection`

Reply:
This is a state that contain some info, like WaitingCandidateInfo.
I do not want to split this into 3 states as it makes maintenance that much harder.